src/sentry/integrations/slack/threads/activity_notifications.py

```diff
@@ -128,6 +128,12 @@ def get_ticket_number(self) -> str:
 
         return ticket_number
 
+    def get_formatted_provider_name(self, provider: str) -> str:
+        # Make sure to make the proper noun have correct capitalization
+        # I.e. github -> GitHub, jira -> Jira
+        # Special cases like github -> GitHub are implemented in their overriden classes
+        return provider.capitalize()
```

Comment:
should not accept a parameter, especially `provider` since this is part of the existing class. We can just do `self.get_provider().capitalize()`